IDE/Renesas/e2studio/DK-S7G2/benchmark-template/src/app_entry.c

```diff
@@ -37,9 +37,9 @@ extern void initialise_monitor_handles(void);
 
 #include "nx_api.h"
 #define CONNECTION_TIMES 100
-#define SERVER_IP IP_ADDRESS(10,22,73,128)
-#define TLS_PORT 11111
-#define TCP_PORT 11112
+#define SERVER_IP IP_ADDRESS(192,168,3,10)
+#define TLS_PORT 11112

```

Comment:
Did you mean to change the default TLS port number? My preference is the TLS one uses 11111 like our example client/server. I see that you also used it this way in the new IDE/Renesas/e2studio/SK-S7G2/README.md docs.